config/writePropertyFile.sh

```diff
@@ -67,6 +67,11 @@ else
     echo "invoker.container.network=bridge" >> "$WHISK_HOME/whisk.properties"
 fi
 
+if [ "$INVOKER_CONTAINER_POLICY" != "" ]; then
+    echo "invoker.container.policy=$INVOKER_CONTAINER_POLICY" >> "$WHISK_HOME/whisk.properties"
+else
+    echo "invoker.container.policy=docker-default" >> "$WHISK_HOME/whisk.properties"
+fi
```

Comment:
I do believe, that with these in place `Map(invokerContainerPolicy -> "")` will never return `""` thus the `Option` will never be `None`. Intentional?